Upstream-Status: Pending

This requires libstdc++.a to be installed on host systems, which is
often not very common, for no particular reason. Still investigating
how to best approach this upstream.

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From af8028b7da98866870d97d965d1e4e85e7db7b1b Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Mon, 17 Sep 2018 16:44:36 +0200
Subject: [PATCH] gen: Stop passing -static-libstdc++ to the compiler

---
 build/gen.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/gen.py b/build/gen.py
index fde6ced2..2a8dd25e 100755
--- a/tools/gn/build/gen.py
+++ b/tools/gn/build/gen.py
@@ -320,7 +320,6 @@ def WriteGNNinja(path, platform, host, options):
 
     if platform.is_linux():
       ldflags.extend([
-          '-static-libstdc++',
           '-Wl,--as-needed',
       ])
       # This is needed by libc++.

-- 
2.17.1

